From 46db262ec1cac6a520a816e6869d7e239c15951e Mon Sep 17 00:00:00 2001
From: Derek Schuff <dschuff@chromium.org>
Date: Thu, 12 Feb 2026 03:15:10 +0000
Subject: [PATCH 18/30] TestLoop: fix signature of main

---
 SingleSource/UnitTests/TestLoop.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SingleSource/UnitTests/TestLoop.c b/SingleSource/UnitTests/TestLoop.c
index a66a92497..c889e3dc7 100644
--- a/SingleSource/UnitTests/TestLoop.c
+++ b/SingleSource/UnitTests/TestLoop.c
@@ -6,7 +6,7 @@ int foo(int i, int j) {
 }
 
 
-int main(int level) {
+int main(int level, char** argv) {
     int i;
     int fval[4];
     //for (i = 3; i < 10; i--)
-- 
2.53.0.1213.gd9a14994de-goog

